From ef586319fbef1c5f9930824b90cb55fe786dcae3 Mon Sep 17 00:00:00 2001
From: Sumit Garg <sumit.garg@linaro.org>
Date: Tue, 15 Jun 2021 17:41:03 +0530
Subject: [PATCH] MAINTAINERS: Add myself as TEE subsystem reviewer

Since I have been helping with TEE subsystem reviews, so make that role
official.

Signed-off-by: Sumit Garg <sumit.garg@linaro.org>
Signed-off-by: Jens Wiklander <jens.wiklander@linaro.org>
---
 MAINTAINERS | 1 +
 1 file changed, 1 insertion(+)

diff --git a/MAINTAINERS b/MAINTAINERS
index 6dca218f2477..41ef24c2501b 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -17188,6 +17188,7 @@ F:	include/media/i2c/tw9910.h
 
 TEE SUBSYSTEM
 M:	Jens Wiklander <jens.wiklander@linaro.org>
+R:	Sumit Garg <sumit.garg@linaro.org>
 L:	op-tee@lists.trustedfirmware.org
 S:	Maintained
 F:	Documentation/staging/tee.rst
